enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,99 @@
+
 const enhancer = require('./enhancer.js');
-// test away!
+
+// TEST ITEM for ALL TESTS
+const elvinBow = {
+    name:'Elvin Bow',
+    durability: 90,
+    enhancement: 0,
+};
+
+describe('enhancer.js', () => {
+    //REPAIR FUNCTION
+    describe('repair(item)', () => {
+        it('should return an object with durability = 100', () => {
+            const {name,enhancement} = elvinBow;
+            expect(enhancer.repair(elvinBow)).toEqual({
+                name: name,
+                durability:100,
+                enhancement: enhancement
+            });
+        });
+    });
+
+    //SUCCEED FUNCTION
+    describe('succeed(item)', () => {
+        const {name,durability,enhancement} = elvinBow;
+        if (enhancement < 20) {
+            it('should return an object with enhancement+1', () => {
+                expect(enhancer.succeed(elvinBow)).toEqual({
+                    name: name,
+                    durability:durability,
+                    enhancement: enhancement+1
+                });
+            });
+        } else {
```

Comment:
ah, so you don't want to actually run else/if statements in your tests -- note that the number of tests jest says your run for this file is only 4 since it will never run both cases of an if/else when only passed one item. instead, either make a new item here with enhancement=20, or reassign elvinBow's enhancement value to 20 after the last test and before this next one to be more declarative